internal/route/routes.go

```diff
@@ -32,9 +31,11 @@ func HandleIndex(w http.ResponseWriter, r *http.Request) {
 			fmt.Fprintf(w, "ParseForm() err: %v", err)
 			return
 		}
-		var todos []string
+		todoList := &model.TodoList{
+			Name: p.Sanitize(r.FormValue("name")),
+		}
+
```

Comment:
In go it's common to use the structs themselves instead of intermediary data types. So in this case we just create a new TodoList struct and put the name in from the form.